arch/arm/boot/dts/overlays/revpi-connect4-overlay.dts

```diff
@@ -0,0 +1,262 @@
+			};
+			exp_power_pins: exp_power_pins {
+				brcm,pins     = <9>;
+				brcm,function = <BCM2835_FSEL_GPIO_IN>;
+				brcm,pull     = <BCM2835_PUD_OFF>;
+			};
+			i2c1_pins: i2c1_pins {
+				/* SDA, SCL */
+				brcm,pins     = <2 3>;
+				brcm,function = <BCM2835_FSEL_ALT0>;
+			};
+			rs485_pins: rs485_pins {
+				/* TX RX TX_EN TERM */
+				brcm,pins     = <12 13 15 14>;
+				brcm,function = <BCM2835_FSEL_ALT4
+				                 BCM2835_FSEL_ALT4
+				                 BCM2835_FSEL_ALT4
+				                 BCM2835_FSEL_GPIO_OUT>;
+				brcm,pull     = <BCM2835_PUD_OFF
+				                 BCM2835_PUD_OFF
+				                 BCM2835_PUD_DOWN
+				                 BCM2835_PUD_DOWN>;
+			};
+			spi1_cs_pins: spi1_cs_pins {
+				/* tpm (SPI1_CE0_N) */
+				brcm,pins     = <18>;
+				brcm,function = <BCM2835_FSEL_GPIO_OUT>;
+			};
+			spi1_pins: spi1_pins {
+				/* MISO, MOSI, SCLK */
+				brcm,pins     = <19 20 21>;
+				brcm,function = <BCM2835_FSEL_ALT4>;
+			};
+			tpm_pins: tpm_pins {
+				/* IRQ */
+				brcm,pins     = <10>;
+				brcm,function = <BCM2835_FSEL_GPIO_IN>;
+			};
+		};
+	};
+
+	/* I2C1 */
+	fragment@2 {
+		target = <&i2c1>;
+		__overlay__ {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			pinctrl-names = "default";
+			pinctrl-0 = <&i2c1_pins>;
+			clock-frequency = <400000>;
+			status = "okay";
+
+			expander_power: gpio@21 {
+				/* 100kHz, 400kHz */
+				compatible = "nxp,pcal6416";
+				pinctrl-names = "default";
+				pinctrl-0 = <&exp_power_pins>;
+				reg = <0x21>;
+				gpio-controller;
+				#gpio-cells = <2>;
+				status = "okay";
+				interrupt-controller;
+				#interrupt-cells = <2>;
+				interrupt-parent = <&gpio>;
+				interrupts = <9 IRQ_TYPE_LEVEL_LOW>;
+			};
+
+			expander_core: gpio@22 {
+				/* 100kHz, 400kHz */
+				compatible = "nxp,pcal6524";
+				pinctrl-names = "default";
+				pinctrl-0 = <&exp_core_pins>;
+				reg = <0x22>;
+				gpio-controller;
+				#gpio-cells = <2>;
+				status = "okay";
+				interrupt-controller;
+				#interrupt-cells = <2>;
+				interrupt-parent = <&gpio>;
+				interrupts = <8 IRQ_TYPE_LEVEL_LOW>;
+			};
+
+			rtc@51 {
+				/* 100kHz, 400kHz */
+				compatible = "nxp,pcf2129";
+				reg = <0x51>;
+				status = "okay";
+				reset-source;
+			};
+
+			/* For system management of PCIe device, not used for now*/
+			pcie@38 {
+				reg = <0x38>;
+				status = "disabled";
+			};
+		};
+	};
+
+	/* SPI1 */
+	fragment@4 {
+		target = <&spi1>;
+		__overlay__ {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			pinctrl-names = "default";
+			pinctrl-0 = <&spi1_pins>, <&spi1_cs_pins>;
+			cs-gpios = <&gpio 18 GPIO_ACTIVE_LOW>;
+			status = "okay";
+
+			tpm: tpm@0 {
+				compatible = "infineon,slb9670";
+				pinctrl-names = "default";
+				pinctrl-0 = <&tpm_pins>;
+				reg = <0>;
+				spi-max-frequency = <1000000>;
+				interrupt-parent = <&gpio>;
+				#interrupt-cells = <2>;
+				interrupts = <10 IRQ_TYPE_LEVEL_LOW>;
+				status = "okay";
+			};
+		};
+	};
+
+	/* UART5 (Front)*/
```

Comment:
The comment is misleading, as we don't have a front uart but an rs485

Reply:
fixed as /* UART5 (RS485)*/